packages/router-generator/src/generator.ts

```diff
@@ -583,7 +583,16 @@ export async function generator(config: Config) {
   interface FileRoutesByPath {
     ${routeNodes
       .map((routeNode) => {
-        return `'${removeTrailingUnderscores(routeNode.routePath)}': {
+        const filePathId = removeTrailingUnderscores(routeNode.routePath)
+        const id = filePathId
+        const fullPath = removeGroups(
```

Comment:
@schiller-manuel Just want to check with you that this logic is correct and I'm not missing something?

Reply:
Checking the "kitchen-sink-file-based" example, currently the `/router-group` route has its id showing up as `/route-group` and NOT `/(this-folder-is-not-in-the-url)/route-group`.

@chorobin not sure if you want the `id` value here to represent the what the route id is at runtime in the router, but if so, you'd probably want to remove the groups here as well.

Reply:
Thanks for catching that. I've updated the tests and the code to remove the route groups